src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -22,13 +22,29 @@ namespace {
 
 namespace crimson::os::seastore::random_block_device::nvme {
 
+NVMeBlockDevice::mkfs_ret NVMeBlockDevice::mkfs(device_config_t config) {
+  using crimson::common::get_conf;
+  shard_devices.local().set_journal_size(
+    get_conf<Option::size_t>("seastore_cbjournal_size"));
+  return shard_devices.local().do_primary_mkfs(config, seastar::smp::count
+  ).safe_then([this] {
+    return shard_devices.invoke_on_all([](auto &local_device) {
```

Comment:
primay shard_info is already get by do_primary_mkfs, so here invoke_on_others should be OK.

Reply:
Done